content/get_start/demo/demo1.md

```diff
@@ -1,96 +1,144 @@
 ---
-title: "Demo 1"
+title: "Demo 1: Geometry"
+alttitle: "De"
 date: 2022-08-16T13:04:42-07:00
 weight: 242
 draft: false
 pre: "<b>- </b>"
 ---
 
-### Demo 1 - Transformation
+{{<line_break>}}
 
-The proposed model is a typical medium sized office building with roughly `60%` window-to-wall ratio (WWR) and overhangs on its south facade.
-![Geometry Transformation Image](/BEM-for-PRM/get_start/os_engine/images/geometry_demo_screenshot.png?width=400px&align=right&classes=border,alignLeft)
-{{< line_break >}}
+#### Demo 1 shows geometry transformations in a user model
 
-#### Project parameters
 
-First, we will need to check the project parameters in the script.
+The proposed model (as shown in the image) is a typical medium sized office building with roughly `60%` window-to-wall ratio (WWR) and overhangs on its south facade. The `default_hvac_bldg_type` is set to `other nonresidential`, and the window to wall ratio and service water heating system building types are both set according to the related category. The building is assumed to be located in New York City. So its climate zone is `4A`.
 
```

Comment:
Add a note that in this demo, we are using a user model as proposed model to simplify the process.